vllm/lora/utils.py

```diff
@@ -100,15 +100,20 @@ def parse_fine_tuned_lora_name(name: str) -> Tuple[str, bool]:
         Tuple(module_name, is_lora_a):
             module_name: the name of the module, e.g. model.dense1,
             is_lora_a whether the tensor is lora_a or lora_b.
+            is_bias whether the tensor is lora bias.
     """
     parts = name.split(".")
+    assert parts[0] == "base_model"
+    assert parts[1] == "model"
+    if parts[-1] == "weight":
+        assert parts[-2] == "lora_A" or parts[-2] == "lora_B"
```

Comment:
This assertion is failing a couple of lora_modules that we have:
```
>>> lora_modules[0].split(".")
['base_model', 'model', 'lm_head', 'weight']
>>> lora_modules[1].split(".")
['base_model', 'model', 'model', 'embed_tokens', 'weight']
```
Still investigating if this would have thrown an error with the previous code as well...

Reply:
@followumesh actually it looks like this was a bad merge .. it's reverting a recent change that was made to improve the error message

Reply:
@prashantgupta24 Can you point to a lora module I can test with?

Reply:
Let me try to find one, also reverting this change gives me the error
```
ValueError: base_model.model.lm_head.weight is unsupported LoRA weight
```

Reply:
To summarize, since vLLM expects only LoRA weights in the safetensors file, it was actually an error on our end that we had lm_head within it. But I think the original error is now being reverted because of this change. Ideally, the error should be
```
ValueError: base_model.model.lm_head.weight is unsupported LoRA weight
```

Instead, an assertion error is now being thrown, which doesn't provide the right detail

Reply:
Yes, @followumesh I think the changes here are not actually related to the main PR changes, wonder if you could revert (and any other changes in same category)?

Reply:
@prashantgupta24 Can you check now?

Reply:
Yeah this looks better. I'll let you know when I get a chance to test it!